PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+- Links to documentation:
+- Link to NPM: 
```

Comment:
Maybe we should prioritize Github repo links to npm. I often found myself checking the code to assert some points.
Also sometimes npm packages don't have a repo link.

Reply:
true!